nearuplib/nodelib.py

```diff
@@ -230,18 +229,13 @@ def setup_and_run(binary_path,
             os.makedirs(binary_path)
 
         download_binaries(chain_id, uname)
-        watch = True
```

Comment:
Removing this won't start watcher anymore or?

Reply:
Ah good catch. It now defaults to `True` and `--no-watcher` will be passed as `not no_watcher` in the `nearup run` command

Reply:
The idea here was to make it line up with the positive assertion that the watcher is running, and simplify the logic here. The `if no_watcher` isn't needed anymore

Reply:
discussed IRL - see: https://github.com/near/nearup/pull/110/commits/ce6977e17d0006c6190b94e8c6f7cb7589116ac8